src/WebApi/Controllers/UsersController.cs

```diff
@@ -210,6 +211,35 @@ public Task<ActionResult<Result<CharacterCharacteristicsViewModel>>> GetCharacte
         }));
     }
 
+    /// <summary>
+    /// Get character rating.
+    /// </summary>
+    /// <param name="id">Character id.</param>
+    /// <returns>The character rating.</returns>
+    /// <response code="200">Ok.</response>
+    [HttpGet("self/characters/{id}/rating")]
+    public Task<ActionResult<Result<CharacterRatingViewModel>>> GetCharacterRating([FromRoute] int id)
+    {
+        return ResultToActionAsync(Mediator.Send(new GetUserCharacterRatingQuery
+        {
+            UserId = CurrentUser.User!.Id,
+            CharacterId = id,
+        }));
+    }
+
+    /// <summary>
+    /// Updates every character competitive rating.
+    /// </summary>>
+    /// <response code="200">Updated.</response>
+    /// <response code="400">Bad Request.</response>
+    [Authorize(AdminPolicy)]
+    [HttpPut("characters/updatecompetitiveratings")]
```

Comment:
Kebab case is often used for URLs, also since the method is a `PUT` the `update` is a bit redundant in the name. What about `PUT characters/competitive-ratings`.

Reply:
done